src/shared/TableEditor/EditableRow.tsx

```diff
@@ -32,141 +40,197 @@ const EditableRow = (props: Props): JSX.Element => {
+            select={true}
+            size="medium"
+            margin="dense"
+            SelectProps={{
+              MenuProps: { disableScrollLock: true }
+            }}
+          >
+            {codes && codes.map((item) => (
+              <MenuItem key={item} value={item}>{item}</MenuItem>
+            ))}
+          </TextField>
+            : props.event}</TableCell>
+        <TableCell>
+          <TextField
+            value={props.year}
+            onChange={(e) => props.onChange(props.index, { key: 'year', value: e.target.value })}
+            select={true}
+            size="medium"
+            margin="dense"
+            SelectProps={{
+              MenuProps: { disableScrollLock: true }
+            }}
+          >
+            {[currentYear - 1, currentYear].map((item) => (
+              <MenuItem key={item} value={item}>{item}</MenuItem>
+            ))}
+          </TextField>
 
-      </TableCell>
-      <TableCell>
-        {props.editable ? <TextField
-          value={props.event}
-          onChange={(e) => props.onChange(props.index, { key: 'event', value: e.target.value })}
-          select={true}
-          size="medium"
-          margin="dense"
-          SelectProps={{
-            MenuProps: { disableScrollLock: true }
-          }}
-        >
-          {codes && codes.map((item) => (
-            <MenuItem key={item} value={item}>{item}</MenuItem>
-          ))}
-        </TextField>
-          : props.event}</TableCell>
-      <TableCell>
-        <TextField
-          value={props.year}
-          onChange={(e) => props.onChange(props.index, { key: 'year', value: e.target.value })}
-          select={true}
-          size="medium"
-          margin="dense"
-          SelectProps={{
-            MenuProps: { disableScrollLock: true }
-          }}
-        >
-          {[currentYear - 1, currentYear].map((item) => (
-            <MenuItem key={item} value={item}>{item}</MenuItem>
-          ))}
-        </TextField>
+        </TableCell>
+        <TableCell>        
+          <TextField
+            value={props.title}
+            onChange={(e) => props.onChange(props.index, { key: 'title', value: e.target.value })}
+            size="medium"
+            margin="dense"
+          /></TableCell>
+        <TableCell>
+          {props.letterReceived ? <Tooltip title="Wpłynęło pismo" aria-label="letter-received">
+            <IconButton><MailIcon/></IconButton>
+          </Tooltip> : <></>}
+          {props.dateOfLetter && props.dateOfLetter !== '' && props.dateOfLetter !== null ? <Tooltip title="Dodano datę pisma" aria-label="date-of-letter">
+            <IconButton><EventIcon/></IconButton>
+          </Tooltip> : <></>}
+          {props.comment && props.comment !== '' ? <Tooltip title="Dodano komentarz" aria-label="added-comment">
+            <IconButton><InsertCommentIcon/></IconButton>
+          </Tooltip> : <></>}
```

Comment:
why you used ternary operation and return empty fragment JSX ? Maybe better to use logical && operation to return element or not return

Reply:
This is the reason of using ternary:  
[https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx/](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx/)

Ofcourse - not at all cases need this method.

Reply:
ok nice, but i still have a question, why return <></>, you not 'null' ?